### DISCUSSION TITLE
v2.13.0

### DISCUSSION THREAD

**Discussion by mikecao** [umami-software/umami#2896] [General]
## Features

This release brings you two new screens, **Events** and **Sessions** giving you more insight into your visitors and activity, custom session data, bug fixes and much more!

### New Events screen

The events screen displays information related to your custom events and custom data. The events chart and metrics table have been moved here from the over overview screen. Below is a new table showing all your event activity and who performed the event. Each avatar represents a unique visitor.

![image](https://github.com/user-attachments/assets/6c1775be-6bd5-4cdb-a393-85c9d7738a85)

The old event data screen has been deprecated. Now your custom data can be accessed under the **Properties** tab on the events screen. This section will show you all the custom data properties you saved as well as a breakdown of all the values. To learn more about custom events, see our [documentation](https://umami.is/docs/track-events).

![image](https://github.com/user-attachments/assets/bcb97d2b-7467-4a4b-b298-4017578db7cc)

### New Sessions screen

The sessions screen displays information about your visitors. We are also introducing a brand new component showing you an hourly traffic breakdown during a given week. Now you can tell when your website is the most busy.

![image](https://github.com/user-attachments/assets/be3cd13b-d494-4b71-b3d3-475c39886c1d)

### Visitor profile

Clicking on any of the avatars in the activity table bring you to a new page showing details about a particular visitor. It also shows their activity history over time.

![image](https://github.com/user-attachments/assets/5675087c-fe8f-4a87-bfcf-403c40816e17)

### Session data

We now support saving session data. It works very similarly to custom event data. You would call the **identity** function from the tracker script to save data about the current session.

```javascript
umami.identify({ email: "bob@aol.com" });
```

Saved session data will appear in the new session profile page under Properties.

![image](https://github.com/user-attachments/assets/42b0a88e-27d7-4b16-b09c-17abb4126349)

## We want your feedback!
Please take 30 seconds to fill out this [short survey](https://docs.google.com/forms/d/e/1FAIpQLScHb1vdxMq7e22Sn-X3WWWnGOSdnm37Ouw-4HoPZIahwEZivA/viewform?usp=sf_link). Thank you!

## Fixes
- Fixed PORT variable being ignored #2880 
- Fixed search box disappearing #2879 

## Updates
- NextJS upgraded to `v14.2.5`
- Prisma upgraded to `v5.17.0`
- Allow team managers to transfer websites to teams #2873 
- Language updates: Catalan (Spain), Korean, Polish, Chinese, German, Khmer, Belarusian, Japanese

## Thank you
@IvanGnusarkov @ccbikai @mho22 @gerardnll @mobeicanyue @RikoDEV @konstantintutsch @Manethpak @Cuzart @eagleon @Feuerhamster @israelozarate7 @beforetech @lsvs @RayeEvtuch @wjsoj @err931 

<hr /><em>This discussion was created from the release <a href='https://github.com/umami-software/umami/releases/tag/v2.13.0'>v2.13.0</a>.</em>

**Comment by Joedmin** [comment]
Hello, thank you very much for developing Umami and this awesome update!
I've today updated to the latest version and wanted to ask if there is any hidden meaning behind the colored dost in each visits in the **Visitor profile** page

**Reply by mikecao** [reply]
Yes, each colored dot represents a visit.

**Reply by Joedmin** [reply]
I see, thank you!
